ee/ucp/admin/install/install-on-azure.md

```diff
@@ -49,56 +131,40 @@ plus a buffer for initial allocations for primary IP addresses.
 > requested.
 {: .important}
 
-## Configure IP pools for Azure stand-alone VMs
+#### Additional Notes
 
-Follow these steps when the cluster is deployed using stand-alone Azure VMs.
+- The `IP_COUNT` variable defines the subnet size for each node's pod IPs. This subnet size is the same for all hosts.
+-  The Kubernetes `pod-cidr` must match the Azure Vnet of the hosts. 
```

Comment:
Not sure if this needs to be the same, or just a subset of the Vnet?

Reply:
Myself and John have chatted offline about this. We will do some more work to test if a subset of the vnet will work fine and if so amend this in a second PR.